### PR TITLE
[Fixes 5467] Updates Sample Scripts

### LINKED ISSUE #5467: Capitalization and consistency.
Please follow consistent capitalization rules in examples. You don't need to be following a [style guide](https://github.com/PoshCode/PowerShellPracticeAndStyle/blob/master/Style-Guide/Code-Layout-and-Formatting.md#capitalization-conventions) to be internally consistent. Using `$textBox` and `$CancelButton` in the same script is confusing.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7c144df0-1934-e637-1fcb-935c799f9013
* Version Independent ID: 37ab8d68-1857-7d5c-03a1-986e82daaa99
* Content: [Creating a Custom Input Box - PowerShell](https://docs.microsoft.com/en-us/powershell/scripting/samples/creating-a-custom-input-box?view=powershell-7&source=docs#feedback)
* Content Source: [reference/docs-conceptual/samples/Creating-a-Custom-Input-Box.md](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/docs-conceptual/samples/Creating-a-Custom-Input-Box.md)
* Product: **powershell**
* Technology: **powershell-conceptual**
* GitHub Login: @joeyaiello
* Microsoft Alias: **jaiello**

### PR DESCRIPTION
**Pull request by chasewilson**
# PR Summary
User noticed inconsistent formatting in documented example scripts. Updated for suggested best practice.

## PR Context
Fixes #5467 
Fixes [AB#1680014](https://dev.azure.com/mseng/677da0fb-b067-4f77-b89b-f32c12bb8617/_workitems/edit/1680014)

Select the type(s) of documents being changed.

**Cmdlet reference & about_ topics**
- [ ] Version 7.x preview content
- [ ] Version 7.0 content
- [ ] Version 6 content
- [ ] Version 5.1 content

**Conceptual articles**
- [ ] Fundamental conceptual articles
- [x] Script sample articles
- [ ] DSC articles
- [ ] Gallery articles
- [ ] JEA articles
- [ ] WMF articles
- [ ] SDK articles

## PR Checklist

- [x] I have read the [contributors guide](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/CONTRIBUTING.md) and followed the style and process guidelines
- [x] PR has a meaningful title
- [x] PR is targeted at the _staging_ branch
- [x] All relevant versions updated
- [x] Includes content related to issues and PRs - see [Closing issues using keywords](https://help.github.com/en/articles/closing-issues-using-keywords).
- [x] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the
    title and remove the prefix when the PR is ready.

### CODE REVIEW

File: reference/docs-conceptual/samples/Creating-a-Custom-Input-Box.md [hunk 1]
```diff
@@ -113,7 +126,10 @@ $label.Text = 'Please enter the information in the space below:'
 $form.Controls.Add($label)
 ```
 
-Add the control (in this case, a text box) that lets users provide the information you’ve described in your label text. There are many other controls you can apply besides text boxes; for more controls, see [System.Windows.Forms Namespace](https://msdn.microsoft.com/library/k50ex0x9(v=vs.110).aspx) on MSDN.
+Add the control (in this case, a text box) that lets users provide the information you’ve described
+in your label text. There are many other controls you can apply besides text boxes; for more
+controls, see [System.Windows.Forms Namespace](https://msdn.microsoft.com/library/k50ex0x9(v=vs.110).aspx)
+on MSDN.
```
Comment (DCtheGeek) [inline reference/docs-conceptual/samples/Creating-a-Custom-Input-Box.md]: Let's fix this link (and `on MSDN` comment) while you are in this article. Update the embedded links at the bottom of the article, also.

File: reference/docs-conceptual/samples/Selecting-Items-from-a-List-Box.md [hunk 2]
```diff
@@ -134,8 +151,10 @@ $listBox.Height = 80
 
 In the next section, you specify the values you want the list box to display to users.
 
-> [!NOTE]
-> The list box created by this script allows only one selection. To create a list box control that allows multiple selections, specify a value for the **SelectionMode** property, similarly to the following:  `$listBox.SelectionMode = 'MultiExtended'`. For more information, see [Multiple-selection List Boxes](Multiple-selection-List-Boxes.md).
+> [!NOTE] The list box created by this script allows only one selection. To create a list box
```
Comment (DCtheGeek) [inline reference/docs-conceptual/samples/Selecting-Items-from-a-List-Box.md]: This note needs fixing too, that's what the validation message is about.

File: reference/docs-conceptual/samples/Creating-a-Custom-Input-Box.md (hunk 1 above)
Comment (chasewilson) [reply reference/docs-conceptual/samples/Creating-a-Custom-Input-Box.md]: Fixed

File: reference/docs-conceptual/samples/Selecting-Items-from-a-List-Box.md (hunk 2 above)
Comment (chasewilson) [reply reference/docs-conceptual/samples/Selecting-Items-from-a-List-Box.md]: Fixed

**Pull request merged**